frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -155,18 +155,30 @@ export const asProperty = (point: PointFeature): IProperty => {
 /**
  * Convert any object to a cql filter string, assuming the object's keys should be used as CQL filter properties.
  * AND all object keys together within the generated cql filter string.
+ *
+ * The resulting filter is URL-encoded
  * @param object an object to convert to a cql filter string.
  */
 export const toCqlFilter = (object: any, pidOverride?: boolean) => {
+  const cqlValue: string = toCqlFilterValue(object, pidOverride);
+  return cqlValue.length ? `cql_filter=${encodeURIComponent(cqlValue)}` : '';
+};
+
+/**
+ * Convert any object to a cql filter string value, assuming the object's keys should be used as CQL filter properties.
+ * NOTE: The resulting string value is left as-is. It will not be URL-encoded
+ * @param object an object to convert to a cql filter string.
+ */
+export const toCqlFilterValue = (object: any, pidOverride?: boolean) => {
```

Comment:
Needed to call this function to get the cql filter value without `cql_filter={value}`